includes/admin/customers/customers.php

```diff
@@ -768,7 +774,7 @@ function edd_customers_view( $customer = null ) {
 			<thead>
 				<tr>
 					<th><?php esc_html_e( 'Email',   'easy-digital-downloads' ); ?></th>
-					<th><?php esc_html_e( 'Actions', 'easy-digital-downloads' ); ?></th>
+					<th><?php esc_html_e( 'Date Added', 'easy-digital-downloads' ); ?></th>
```

Comment:
The CSS for this table also needs to be updated for responsiveness, with the new column being added.